docs/running.rst

```diff
@@ -126,19 +128,25 @@ Alternatively, to run the simulation in MPI mode:
 
 .. code-block:: none
 
-   $ yank script --yaml=yank.yaml
+   $ mpirun -n # yank script --yaml=yank.yaml
```

Comment:
Let's not use `#` here, and this may not even be necessary. If you're using the conda-installed `mpirun` (hydra), it will autodetect the queue manager and number of processes, won't it?